pcmanfm/desktopentrydialog.ui

```diff
@@ -58,6 +61,11 @@
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="descriptionEdit">
+       <property name="whatsThis">
+        <string>A short description or generic name, like &quot;Web Browser&quot; or &quot;Home Directory&quot;.
+
```

Comment:
"It will be shown as tooltip" ?

Reply:
Not necessarily. It depends on the DE and its main menu.